js-sdk/integrations/i18next/api.mdx

```diff
@@ -11,35 +11,35 @@ description: 'API i18next: full integration which ensures complete integration o
 
 Ensures complete integration between i18next and Tolgee.
 
-- `i18next` - i18next instance
-- `tolgee` - [`TolgeeInstance`](/api/core_package/tolgee.mdx)
+- `i18next` - the i18next instance
+- `tolgeeConfig` - A [`TolgeeInstance`](/api/core_package/tolgee.mdx)
```

Comment:
`tolgeeConfig` is not a good name, could you please keep the `tolgee` and also fix it in the example above?